checks/code_review.go

```diff
@@ -37,7 +37,7 @@ func init() {
 
 // CodeReview will check if the maintainers perform code review.
 func CodeReview(c *checker.CheckRequest) checker.CheckResult {
-	rawData, err := raw.CodeReview(c.RepoClient)
+	rawData, err := raw.CodeReview(c.RepoClient, c.Dlogger)
```

Comment:
Can we avoid passing the logger here? The raw result collects data only, it should be the evaluator's job to create the final detail information. Also note that raw results don't use the Details in https://github.com/ossf/scorecard/blob/main/pkg/json_raw_results.go